File: backend/src/main/java/com/example/backend/Model/Dto/UserInformationDTO.java
package com.example.backend.Model.Dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserInformationDTO {
    private Long countProcessLog;
    private Long countCars;
    private Long countQualityChecks;
    private Long countAssignedParts;
    private Long countMachineUsed;
    private Long countByPartProduction;

    //Page
    private Long countByMachine;

    //All
    private Long countAllUsers;
    private Long  countAllMachine;
    private Long countAllCars;
    private Long countAllCarModels;
}

File: frontend/src/app/_model/_dto/user-information-dto.ts
export interface UserInformationDTO {
  countProcessLog: number;
  countCars: number;
  countQualityChecks: number;
  countAssignedParts: number;
  countMachineUsed: number;
  countByPartProduction: number;
  //Page
  countByMachine: number;

  //All
  countAllUsers: number;
  countAllMachine: number;
  countAllCars: number;
  countAllCarModels: number;
}
